annsa/annsa.py

```diff
@@ -1,273 +1,46 @@
 from __future__ import print_function
-import numpy as np
-import random
-import datetime
-import os
-from scipy.ndimage.interpolation import zoom
-import matplotlib.pyplot as plt
 
 
-def write_time_and_date():
```

Comment:
Where did all of this deleted code end up?

Reply:
I didn't save them anywhere, I outright deleted them. These functions were used in 2+ year old code.

Reply:
2 years isn't so long.
Are these necessary to reproduce anything in your thesis?

Reply:
No, they are outdated and not used in any of my dissertation work.